PureBasicIDE/AssemblyOperandsX86.pb

```diff
@@ -1,201 +1,289 @@
 ﻿; This file is autogenerated, as the compiler side needs to be updated the same way
-;
-
-Data.l 393
-Data$ "AAA"
-Data$ "AAD"
-Data$ "AAM"
-Data$ "AAS"
-Data$ "ADC"
-Data$ "ADD"
-Data$ "AND"
-Data$ "ARPL"
-Data$ "BOUND"
-Data$ "BSF"
-Data$ "BSR"
-Data$ "BSWAP"
-Data$ "BT"
-Data$ "BTC"
-Data$ "BTR"
-Data$ "BTS"
-Data$ "CALL"
-Data$ "CBW"
-Data$ "CDQ"
-Data$ "CLC"
-Data$ "CLD"
-Data$ "CLI"
-Data$ "CLTS"
-Data$ "CMC"
-Data$ "CMOVA"
-Data$ "CMOVAE"
-Data$ "CMOVB"
-Data$ "CMOVBE"
-Data$ "CMOVC"
-Data$ "CMOVE"
-Data$ "CMOVG"
-Data$ "CMOVGE"
-Data$ "CMOVL"
-Data$ "CMOVLE"
-Data$ "CMOVNA"
-Data$ "CMOVNAE"
-Data$ "CMOVNB"
-Data$ "CMOVNBE"
-Data$ "CMOVNC"
-Data$ "CMOVNE"
-Data$ "CMOVNG"
-Data$ "CMOVNGE"
-Data$ "CMOVNL"
-Data$ "CMOVNLE"
-Data$ "CMOVNO"
-Data$ "CMOVNP"
-Data$ "CMOVNS"
-Data$ "CMOVNZ"
-Data$ "CMOVO"
-Data$ "CMOVP"
-Data$ "CMOVPE"
-Data$ "CMOVPO"
-Data$ "CMOVS"
-Data$ "CMOVZ"
-Data$ "CMP"
-Data$ "CMPS"
-Data$ "CMPSB"
-Data$ "CMPSD"
-Data$ "CMPSW"
-Data$ "CMPXCHG"
-Data$ "CMPXCHG8B"
-Data$ "CWD"
-Data$ "CWDE"
-Data$ "DAA"
-Data$ "DAS"
-Data$ "DB"
-Data$ "DD"
-Data$ "DEC"
-Data$ "DIV"
-Data$ "DW"
-Data$ "EMMS"
-Data$ "ENTER"
-Data$ "ESC"
-Data$ "F2XM1"
-Data$ "FABS"
-Data$ "FADD"
-Data$ "FADDP"
-Data$ "FBLD"
-Data$ "FBSTP"
-Data$ "FCHS"
-Data$ "FCLEX"
+Data.l 1045
```

Comment:
Probably they should have been on a commit of their own, but since the PR wasn't squashed it's not a big deal anyway.

My only worry is the initial comment in the file, which indicated that it's autogenerated:

    ; This file is autogenerated, as the compiler side needs to be updated the same way

So chances are that it could be written at any time when the PBCompiler is rebuild (on Fred's side). 

Maybe these changes should be integrated into source file that generated this one — probably not part of this repo, but Fred could handle that on his machine.

I personally like that more ASM keywords were added, and really appreciate idle-PB added comments to the instructions,, which are an added benefit. So I hope this file won't get overriden.

There are various files in this repo that are autogenerated; it's worth mentioning them in the READMEs (both in the main README file as well as in each folder where they are found).

Also, autogenerated files are not granted to pass the Travis build (in fact, I had to manually reformat all sources via the PB IDE to ensure the build passed), so some of them might fail the build when re-generated. It might be worth excluding them from the checks if that's the case.